docs/workloads/azure/data/components/etl_pipelines/silver_data_azure.md

```diff
@@ -0,0 +1,45 @@
+# Silver processing
+
+Silver processing represents the 'Bronze to Silver' layer transformations. It accepts raw data
+(e.g. CSV files) as an input and converts it into a more structured delta format.
```

Comment:
Maybe need to discuss this... Bronze data may already be in Parquet format (if ADF ingest loads it in this way). Further bespoke transformations will often need to happen at this layer too.

Reply:
Right now the documentation reflects exactly what our code currently does. I'm open to extending the scope of silver layer, but I'm not sure what are the requirements.